app/policies/collection_policy.rb

```diff
@@ -7,11 +7,19 @@ class CollectionPolicy < ApplicationPolicy
     relation.where(creator: user)
   end
 
+  alias_rule :edit?, to: :update?
+
   # Those who are members of the LDAP collection group may create collections
   sig { returns(T::Boolean) }
   def create?
     administrator? || collection_creator?
   end
 
+  # TODO: don't allow everyone to update collections
+  sig { returns(T::Boolean) }
+  def update?
+    true
```

Comment:
Is this still right? Or should we do the same checks as in `create?`

Reply:
That's a separate ticket, #254